src/main/java/com/digitax/exception/ErrorExceptionDetail.java

```diff
@@ -0,0 +1,27 @@
+package com.digitax.exception;
+
+import javax.xml.ws.WebFault;
+
+import gov.irs.a2a.mef.mefheader.ErrorExceptionDetailType;
+
+@WebFault(name = "ErrorExceptionDetail", targetNamespace = "http://www.irs.gov/a2a/mef/MeFHeader.xsd")
+public class ErrorExceptionDetail extends Exception{
+	
+	private ErrorExceptionDetailType faultInfo;	
+	public ErrorExceptionDetail(String message, ErrorExceptionDetailType faultInfo) {
```

Comment:
@Gyana-Redapple Be mindful of formatting, inline comments, method comments and class comments. You are missing all of these things.